docs/Configuration/BitBucket.md

```diff
@@ -47,20 +47,88 @@ If you are behind a corporate firewall or VPN you may need to utilize a proxy se
 
 DevLake uses a dynamic rate limit to collect BitBucket data. You can adjust the rate limit if you want to increase or lower the speed.
 
-The maximum rate limit for different entities in BitBucket(Cloud) is [60,000 or 1,000 requests/hour](https://support.atlassian.com/bitbucket-cloud/docs/api-request-limits/). Please do not use a rate that exceeds this number.
+The maximum rate limit for different entities in BitBucket(Cloud) is [1,000 requests/hour](https://support.atlassian.com/BitBucket-cloud/docs/api-request-limits/). But we find it can run when we try a rate limit of more than 1000/h. So you can try the bigger maximum if your repo is big enough.
+
+<!-- ![image](https://user-images.githubusercontent.com/3294100/220094172-9e8e9e8b-75ea-4c3e-8e5b-716320dabb64.png) -->
 
 
 #### Test and Save Connection
 
 Click `Test Connection`, if the connection is successful, click `Save Connection` to add the connection.
 
-
 ### Step 2 - Configure Blueprint
 
-Similar to other beta plugins, BitBucket does not support `project`, which means, you can only collect BitBucket data via blueprint's advanced mode. 
+![image](https://user-images.githubusercontent.com/3294100/220236338-772b30b7-974f-4bc2-89ce-f2abe5e92a5e.png)
+
+#### Repositories
+
+Select the BitBucket repos to collect.
+
+#### Data Entities
+
+Usually, you don't have to modify this part. However, if you don't want to collect certain BitBucket entities, you can unselect some entities to accelerate the collection speed.
+
+- Issue Tracking: BitBucket issues, issue comments, etc.
```

Comment:
Have we collected BitBucket issue comments? If not, delete it.

Reply:
Yes, 
![image](https://user-images.githubusercontent.com/3294100/221530002-e082de66-8768-4f4b-b1ce-3255877cf7cf.png)